include/glow/Backends/BackendUtils.h

```diff
@@ -21,6 +21,11 @@
 
 namespace glow {
 namespace runtime {
+
+/// An enum to indicate what type each symbol in the bundle is. Current options
+/// are activations, constants, tensor views, and placeholders.
+enum class SymbolCategory { Activation, Placeholder, Constant, TensorView };
```

Comment:
I'm not sure we need a dedicated category for TensorViews. After all they are derivatives of their underlying symbols. It seems like tensorview symbols should get the same category as their parent symbols.